crates/turborepo-auth/src/lib.rs

```diff
@@ -89,6 +89,30 @@ pub async fn read_or_create_auth_file(
     Ok(auth_file)
 }
 
+/// Sync version of `read_or_create_auth_file`.
+pub fn read_auth_file(auth_file_path: &AbsoluteSystemPath) -> Result<AuthFile, Error> {
+    if auth_file_path.try_exists()? {
+        let content = auth_file_path
+            .read_to_string()
+            .map_err(|e| Error::FailedToReadAuthFile {
+                source: e,
+                path: auth_file_path.to_owned(),
+            })?;
+        let tokens: AuthFile = serde_json::from_str(&content)
+            .map_err(|e| Error::FailedToDeserializeAuthFile { source: e })?;
+        let mut auth_file = AuthFile::new();
+        for (api, token) in tokens.tokens() {
+            auth_file.insert(api.to_owned(), token.to_owned());
+        }
```

Comment:
I don't understand why we're rebuilding the auth file here

Reply:
Lazy copy-paste job - I'll fix that in a moment.